src/styles/button.css

```diff
@@ -1 +1,68 @@
 /* button */
+
+.ui.button {
+  background-color: var(--primary-pms-362_900);
+  color: var(--shade-white);
+  font-family: var(--gotham);
```

Comment:
Do we need this? I'm just thinking if it gets inherited from _body_

Reply:
still here

Reply:
It looks like we _do_ need it because `.ui.button` applies its own `font-family` (Lato), which takes precedence over the one on `body`, so it won't be inherited.